README.md

```diff
@@ -11,7 +14,7 @@ The program works in the following way:
   1. Looks up the best pizza restaurant in that city accoring to the Google Maps Places API
   1. Stores the restaurant data in Firestore
 
-TODO: Add a program diagram that explains how the application works (after architecture is finalized).
+<img width="780" alt="Application Architecture" src="https://user-images.githubusercontent.com/744973/67051965-5c2a1080-f102-11e9-9209-5dcff36e3f8e.png">
```

Comment:
Can you add the product names under the hexes? Also do you want to show the function's call to the GMP api? Also, it might be helpful to add the endpoint labels if not busy, e.g. adding `/target` for the go between tasks and functions.

Reply:
> - Can you add the product names under the hexes?

Done.

> - Also do you want to show the function's call to the GMP api?

Good point. Added GMP API logo.

> - Also, it might be helpful to add the endpoint labels if not busy, e.g. adding /target for the go between tasks and functions.

The visual is at maximum business imo. I hope the developer looks at the routes to understand a bit more of all the URL routes for this application.